appendices/common-arguments.md

```diff
@@ -0,0 +1,131 @@
+
+## errorjump 
+
+The `errorjump` argument specifies name of a label to jump to when an error occurs during command’s execution. It allows G1ANT.Robot to skip some part of the script and jump to the other.
+
+### Example 1
+
+In the example below some text is assigned to the `♥text` variable.
+
+The condition `⊂♥text == "Chris likes bananas"⊃` is false, so, because the `errorjump` argument was specified,  G1ANT.Robot jumps to `label banana` — the value for the `errorjump` argument. G1ANT.Robot will not show dialog with “*Yes! Chris likes bananas!*” message, because the `errorjump` argument is executed before any other command. Therefore, anything within the `banana` label will be executed. In this particular case a dialog message “*Sorry, Chris doesn't like bananas!*” is shown.
+
+```G1ANT
+♥text = ‴Chris likes apples‴
+test condition ⊂♥text == "Chris likes bananas"⊃ errorjump banana
+dialog ‴Yes! Chris likes bananas!‴
+label banana
+dialog ‴Sorry, Chris doesn't like bananas!‴
+```
+
+### Example 2
+
+```G1ANT
+color.expected position 616⫽384 color 1E1E1F errorjump wrong result ♥name
+dialog ♥name
+jump end
+label wrong 
+dialog message ‴no expected color in this position‴
+label end
+```
+
+In this example, if the specified color 1E1E1F is found in the specified position 616⫽384 on the screen, G1ANT.Robot will show a dialog with the contents of the `♥name` variable and then will jump to the `end` label. If the given color is not there, the robot will jump to the `wrong` label, displaying a dialog message: “*no expected color in this position*”.
+
+## errormessage
+
+The `errormessage` argument is a message that will be shown in case an error occurs and no `errorjump` argument is specified. If `errorjump` is specified, G1ANT.Robot will skip the `errormessage` argument while executing the script.
+
+### Example
+
+```G1ANT
+color.expected position 616⫽384 color 1E1E1F errormessage ‴no expected color in this position‴
+```
+
+If a given color is not found, a message “*no expected color in this position*” will be displayed.
+
+## errorresult
+
+This argument allows assigning a variable, which will store the information about the exception that occurred when a command was executed. This information is contained in an [error structure](../indexes/structures/error.md) with `type` and `message` indexes.
+
+### Example
+
+```G1ANT
+window blah errorresult ♥someError errorcall ErrorOccurred
+keyboard OK!
+
+procedure ErrorOccurred
+    dialog ‴♥someError⟦type⟧ error occured: ♥someError⟦message⟧‴
+end procedure
+```
+
+This extended version of the `errorcall` argument example shows a dialog message with a detailed information about the error.
+
+## timeout
+
+The `timeout` argument lets choose the amount of time (in milliseconds) for the robot to wait for a command to be executed before throwing an error about time expiration.
```

Comment:
is it correct? 'lets choose' instead of 'lets us choose'?

Reply:
```suggestion
The `timeout` argument allows to choose the amount of time (in milliseconds) for the robot to wait for a command to be executed before throwing an error about time expiration.
```